pipelines/repeat/bin/repeatmasker.py

```diff
@@ -0,0 +1,289 @@
+# See the NOTICE file distributed with this work for additional information #pylint: disable=missing-module-docstring
```

Comment:
I think is easier calling the module from ensembl-anno so we can keep it updated just there ( I fixed it for the repeatmasker classes recently in the repo)